units/host_allocator.go

```diff
@@ -305,7 +306,7 @@ func (j *hostAllocatorJob) setTargetAndTerminate(ctx context.Context, numUpHosts
 			DistroID:     distro.Id,
 			NewCapTarget: newCapTarget,
 		}
-		err := amboy.EnqueueUniqueJob(ctx, j.env.RemoteQueue(), NewHostDrawdownJob(j.env, drawdownInfo, utility.RoundPartOfMinute(1).Format(TSFormat)))
+		err := amboy.EnqueueUniqueJob(ctx, j.env.RemoteQueue(), NewHostDrawdownJob(j.env, drawdownInfo, utility.RoundPartOfMinute(1).Format(TSFormat)+distro.Id))
```

Comment:
an alternative to doing it this way is to encorporate the distro id in the id and scope [here](https://github.com/evergreen-ci/evergreen/blob/db3f306dac84b094a7a55942a0c6f03995a916a3/units/host_drawdown.go#L67-L68)

Reply:
This definitely _works_, but I think it's more typical to assemble the id in the job's "constructor". Doing it there has the advantage that the id is consistent no matter where the job is created. 
A couple of examples [here](https://github.com/evergreen-ci/evergreen/blob/ea2993ee93fc6fc77fd6dbc6ef33063f74d6ec4f/units/host_allocator.go#L62), [here](https://github.com/evergreen-ci/evergreen/blob/ea2993ee93fc6fc77fd6dbc6ef33063f74d6ec4f/units/scheduler.go#L50), [here](https://github.com/evergreen-ci/evergreen/blob/ea2993ee93fc6fc77fd6dbc6ef33063f74d6ec4f/units/provisioning_user_data_done.go#L64), etc.

Reply:
I changed it to fit this format